components/pedigree/resources/src/main/resources/pedigree/view/personHoverbox.js

```diff
@@ -294,7 +388,14 @@ define([
                 }
             }
             this.animateHideHoverZone();
-        }
+        },
+
+        generateCircle: function(x, y, tick) {
+            if (tick)
+                return editor.getPaper().circle(x, y, 5).attr({"fill": "#000"});
+            else
+                return  editor.getPaper().circle(x, y, 5).attr({'stroke': '#000', 'stroke-width': '1', "fill": "#fff"});
```

Comment:
attributes should be added to PedigreeEditorParameters.attributes instead of hardcoding